pre/remap_restart/remap_bin2nc.py

```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env python
```

Comment:
@weiyuan-jiang Can you test this with:
```
#!/usr/bin/env python3
```
? I've been on a few systems that don't provide a `python` weirdly...or it points to `python2`.